src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -214,7 +213,7 @@ sharded_store::has_schema(const canonical_schema& schema) {
 }
 
 ss::future<canonical_schema_definition>
-sharded_store::get_schema_definition(const schema_id& id) {
+sharded_store::get_schema_definition(schema_id id) {
```

Comment:
great clean up. better for coroutines, easier to analyze lifetimes. very little performance hit.